tests/test_ckan_to_frictionless.py

```diff
@@ -169,56 +169,82 @@ def test_unjsonify_all_extra_values(self):
         assert out == exp
 
     def test_dataset_license(self):
+        # No license_title nor license_url
         indict = {
             'license_id': 'odc-odbl'
         }
-        exp = {
-            'licenses': [{
-                'name': 'odc-odbl'
-            }]
-        }
-        out = converter.dataset(indict)
-        assert out == exp
-
-        indict = {
-            'license_id': 'odc-odbl',
-            'license_title': 'Open Data Commons Open Database License'
-        }
         exp = {
             'licenses': [{
                 'name': 'odc-odbl',
-                'title': 'Open Data Commons Open Database License'
             }]
         }
         out = converter.dataset(indict)
         assert out == exp
 
-        # finally what if license*s* already there ...
+        # Remap everything in licenses
         indict = {
-            'licenses': [{
-                'name': 'odc-pddl'
-            }],
-            'license_id': 'odc-odbl'
+            'license_id': 'cc-by',
+            'license_title': 'Creative Commons Attribution',
+            'license_url': 'http://www.opendefinition.org/licenses/cc-by'
         }
         exp = {
             'licenses': [{
-                'name': 'odc-pddl'
+                'name': 'cc-by',
+                'title': 'Creative Commons Attribution',
+                'path': 'http://www.opendefinition.org/licenses/cc-by'
             }]
         }
         out = converter.dataset(indict)
         assert out == exp
 
     def test_dataset_license_with_licenses_in_extras(self):
+        # If CKAN 'extras' contain a list of 'licenses', it must come
+        # from Frictionless.
         indict = {
             'license_id': 'odc-odbl',
-            # TODO: check package_show in CKAN
-            # 'license_title': 'Open Data Commons Open Database License',
-            # 'license_url': 'http://opendefinition.org/licenses/odc-odbl/'
+            'license_title': 'Open Data Commons Open Database License',
+            'license_url': 'https://opendatacommons.org/licenses/odbl/1-0/index.html',
+            'extras': [
+                {
+                    'key': 'licenses',
+                    'value': json.dumps(
+                        {
+                            'licenses': [
+                                {
+                                    'name': 'cc-by',
+                                    'title': 'Creative Commons Attribution',
+                                    'path': 'http://www.opendefinition.org/licenses/cc-by'
+                                },
+                                {
+                                    'name': 'odc-odbl',
+                                    'title': 'Open Data Commons Open Database License',
+                                    'path': 'https://opendatacommons.org/licenses/odbl/1-0/index.html'
+                                }
```

Comment:
That's a bunch of symbols below, but I think it's still readable enough for this PR in that following the data with the indentation is not too painful. We can refactor if desired (I wanted to avoid hiding stuff away in fixtures. It kills the Python magic but I guess it's better that way :smile:).